resources/etc/nginx/nginx.conf.tpl

```diff
@@ -22,6 +22,12 @@ http {
   include       /etc/nginx/include.d/mime.types;
   default_type  application/octet-stream;
 
+  # Increase proxy buffers to avoid 502 - bad gateway errors in e.g. EasyRedmine.
+  # These are caused by long headers which do not fit into the default buffer size.
+  proxy_busy_buffers_size   512k;
+  proxy_buffers   4 512k;
+  proxy_buffer_size   256k;
+
```

Comment:
Where do these numbers come from? They seem huge compared to the default values (4k-8k). Are there any negative effects of using such big numbers here? Would it be possible to set these numbers only for EasyRedmine? It's for now the only app that has these problems.

Reply:
A quick look at this article recommends only setting the proxy_buffer_size and not to such a high value. Did you try that? https://www.getpagespeed.com/server-setup/nginx/tuning-proxy_buffer_size-in-nginx Maybe we should hav a look, if other values solve the problem as well or am i a little overcautious here?